scripts/pi-hole/php/savesettings.php

```diff
@@ -352,6 +352,13 @@ function addStaticDHCPLease($mac, $ip, $hostname) {
 				}
 				pihole_execute("-a -i ".$DNSinterface." -web");
 
+				// Add rate-limiting settings
+				if(isset($_POST["rate_limit_count"]) && isset($_POST["rate_limit_interval"]))
+				{
+					// Restart of FTL is delayed
+					pihole_execute("-a ratelimit " . intval($_POST["rate_limit_count"]) . " " . intval($_POST["rate_limit_interval"]) . " false");
```

Comment:
Why do you pass `false` here? Shut the setting not take effect immediately?

Reply:
With the current design of the settings page it is not possible to set/change the rate-limiting values alone. Only a few lines down we call `pihole -a setdns ...` which will inevitably restart FTL. No need for two restarts in a short time.